.github/workflows/pull_request.yml

```
@@ -80,3 +80,20 @@ jobs:
           platforms: linux/amd64, linux/arm64, linux/arm/v7, linux/ppc64le
           tags: |
             openebs/provisioner-localpv:ci
+
+  ansible:
```

```suggestion
  localpv-e2e:
```
Could you use the name 'localpv-e2e' here.
For the openebs/provisioner-localpv image the workflow is named 'provisioner-localpv'
I feel it makes sense to use the 'localpv-e2e' name for the openebs/localpv-e2e image.